graphics/openimageio/files/patch-2.4.5-cmake-disable-wall-werror.diff

```diff
@@ -1,26 +0,0 @@
---- src/cmake/compiler.cmake.orig	2023-06-26 09:39:08.000000000 -0400
```

Comment:
Are you sure this patch isn't needed anymore?

See ticket: https://trac.macports.org/ticket/67346

Reply:
Yeah, `-Werror` isn't used in the build. If you check the CMake file you can see that it's only enabled in development builds.

Reply:
> Yeah, `-Werror` isn't used in the build. If you check the CMake file you can see that it's only enabled in development builds.

`-Werror` isn't the only concern, as `-Wall` is still enabled unconditionally. And that's another key concern raised in the ticket.

So at the very least, I'd ask that you not delete the patch. And rather than removing the line from the portfile, simply comment it out for now.

Reply:
The issue in 2.4.5.0 was that the `${PROJECT_NAME}_SUPPORTED_RELEASE` variable was set incorrectly which resulted in `STOP_ON_WARNING` always being on. This was fixed in https://github.com/AcademySoftwareFoundation/OpenImageIO/commit/efd27a9dfb70ae6e6502407bd08f6c141a40854c so there's no need for the patch anymore. `-Wall` doesn't cause any issues and I don't see it mentioned in the ticket. `-Wunknown-warning-option` is also completely unrelated.